test/unit/ngsiv2/expressions/jexlBasedTransformations-test.js

```diff
@@ -385,40 +385,41 @@ describe('Java expression language (JEXL) based transformations plugin', functio
         });
     });
 
-    describe('When an update comes for attributes with string expression and type integer', function () {
-        // Case: Update for an integer attribute with string expression
-        const values = [
-            {
-                name: 'e',
-                type: 'Number',
-                value: 52
-            }
-        ];
-
-        beforeEach(function () {
-            nock.cleanAll();
-
-            contextBrokerMock = nock('http://192.168.1.1:1026')
-                .matchHeader('fiware-service', 'smartgondor')
-                .matchHeader('fiware-servicepath', 'gardens')
-                .post(
-                    '/v2/entities/ws1/attrs',
-                    utils.readExampleFile(
-                        './test/unit/ngsiv2/examples/contextRequests/updateContextExpressionPlugin11.json'
-                    )
-                )
-                .query({ type: 'WeatherStation' })
-                .reply(204);
-        });
-
-        it('should apply the expression before sending the values', function (done) {
-            iotAgentLib.update('ws1', 'WeatherStationMultiple', '', values, function (error) {
-                should.not.exist(error);
-                contextBrokerMock.done();
-                done();
-            });
-        });
-    });
+    // Deleted test by Issue #1034
```

Comment:
If the "autocast after jexl processing" removal is definitive, maybe it's better to remove these tests instead of commenting. What do you think?

Reply:
Fixed in 086a53f